indra/databases/hgnc_client.py

```diff
@@ -268,6 +269,25 @@ def get_hgnc_entry(hgnc_id):
     return xml_tree
 
 
+def get_gene_type(hgnc_id: str) -> Union[str, None]:
```

Comment:
`Union[str, None]` is the same as `Optional[str]`

Reply:
Right, I was thinking, Optional sounds more intelligible for inputs parameters and Union for return values so went with this.

Reply:
Now that you mention it, I actually really like this